gateway/1.6/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -126,6 +126,11 @@ Replace `<your-username>` and `<your-password>` with Anypoint Platform user cred
 +
 Replace `<path-to-registration-file-in-container>` with the path and filename of the existing `registration.yaml` file. You must specify the file in the container file system, which in this case is `/registration/registration.yaml`.
 ====
```

Comment:
Is there an extra ==== on line 128? When I view this file, I don't see the note at all.

Reply:
Hey, that's for the tabs, not the note.